.editorconfig

```diff
@@ -0,0 +1,6 @@
+# http://editorconfig.org
```

Comment:
Just wonder, does `.editorconfig` popular in JS world? It's rarely used in Python or any other world. :)

Reply:
Sure) You can see some of them [here](https://github.com/editorconfig/editorconfig/wiki/Projects-Using-EditorConfig) (eg react, webpack etc.)